client/src/components/ProtectedRoute.js

```diff
@@ -12,14 +12,9 @@ const ProtectedRoute = ({component: Component,...rest}) => {
                     return <Component {...props} />;
                 } else {
                     return (
-                        <Redirect
-                            to={{
-                                pathname: "/login",
-                                state: {
-                                    from: props.location
-                                }
-                            }}
-                        />
+                        <Redirect to='/login'/>,
+                        <DialogTitle id="alert-dialog-title">{"Sie müssen eingeloggt sein!"}</DialogTitle>
+
```

Comment:
without auth you will have an error message like "Sie müssen eingeloggt sein!" by trying to click to radar links in nav bar